.github/workflows/main.yml

```
@@ -97,15 +119,18 @@ jobs:
         tags: ${{ secrets.DOCKERHUB_USERNAME }}/${{ env.DockerImageName }}:${{ github.sha }}
 
   helm-ci:
-    runs-on: ubuntu-latest
+    name: Helm CI
+    runs-on: ${{ github.event_name == 'workflow_dispatch' && github.event.inputs.OsType || 'ubuntu-latest' }}
+    env:
+      OsType: ${{ github.event_name == 'workflow_dispatch' && github.event.inputs.OsType || 'ubuntu-latest' }}
+      PythonVersion: ${{ github.event_name == 'workflow_dispatch' && github.event.inputs.PythonVersion || '3.8' }}
```

Тут дефолтні значення будуть передаватись автоматично. Достатньо викорситати таку конструкцію. Це потрібно ви правити в усіх місцях.
```suggestion
      OsType: ${{github.event.inputs.OsType}}
      PythonVersion: ${{github.event.inputs.PythonVersion}}
```